### ISSUE TITLE
Binding to existent DOM?

### ISSUE THREAD

**Issue by graste** [ractivejs/ractive#981]
Hi,
I really like what I see so far in the docs and examples. I couldn't answer one thing from the existing examples though: Is it possible to bind to an existing DOM? I don't want to create a javascript application, but use ractive to progressively enhance the existing html that was created server-side. Doing the initial rendering client side while the response could just deliver all the necessary html for the initial view would be a waste imho - I don't have good experience trying e.g. knockoutjs using this. It feels like taking ages before seeing the first stuff on the page while all the data is already within the page as json etc.

TL;DR: Can I use ractive two-way binding via binding to an existent DOM? That would be awesome. :>
